Lab4.py

```diff
@@ -177,7 +175,7 @@ def plot_kanger(xmax=100, tmax=1825, dx=0.1, dt=0.2):
     #Only plotting every ten points in order to conserve computer memory:
     plt.pcolor((time/365)[::10], position[::10], temp[::10, ::10], cmap='seismic', vmin=-25, vmax=25)
     plt.gca().invert_yaxis()  # This will invert the y-axis
-    plt.title("Ground Temperature: Kangerlussuaq, Greenland")
+    plt.title(f'Ground Temperature: Kangerlussuaq, Greenland (Temp shift={kanger_cc}C)')
```

Comment:
It would be preferred if the plot title said Temperature Shift instead of Temp shift